python/tvm/relax/training/loss.py

```diff
@@ -0,0 +1,240 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=redefined-builtin, invalid-name
+"""Loss functions library for relax."""
+
+from typing import Optional, Union
+from tvm import relax
+from ..expr import Expr, Var, Function, StructInfo
+
+from ..op import abs, sum, mean, subtract, multiply
+from ..op.nn import log_softmax, nll_loss
+
+
+__all__ = ["L1Loss", "MSELoss", "CrossEntropyLoss"]
+
+
+def _create_param_var(param: Union[Var, StructInfo], param_name) -> Var:
+    if isinstance(param, StructInfo):
+        param = Var(param_name, param)
+    assert isinstance(param, Var)
+    return param
+
+
+class Loss:
+    """Base class of all loss.
+
+    Parameters
+    ----------
+    loss_name : str
```

Comment:
Does `Literal["l1_loss", "mse_loss", "cross_entropy_loss"]` look strange since we will have more losses in the future, and the annotation will be very long?